include/netlink/route/link/bonding.h

```diff
@@ -26,10 +26,22 @@ extern int	rtnl_link_bond_release_ifindex(struct nl_sock *, int);
 extern int	rtnl_link_bond_release(struct nl_sock *, struct rtnl_link *);
 
 extern void	rtnl_link_bond_set_mode(struct rtnl_link *link, uint8_t mode);
+extern uint8_t	rtnl_link_bond_get_mode(struct rtnl_link *link);
+
 extern void	rtnl_link_bond_set_activeslave(struct rtnl_link *link, int active_slave);
+extern int	rtnl_link_bond_get_activeslave(struct rtnl_link *link);
```

Comment:
The usual way for these is to return an int, and store the value in a pointer passed as argument, so you can actually return error values if the value isn't set.

```suggestion
extern int	rtnl_link_bond_get_activeslave(struct rtnl_link *link, int *active_slave);
```

E.g. activeslave wouldn't exist for an LACP bond, so we should return an error here.